web/src/components/__tests__/netflow-traffic.spec.tsx

```diff
@@ -1,13 +1,30 @@
 import * as React from 'react';
-import { shallow } from 'enzyme';
-
+import { shallow, render } from 'enzyme';
+import { useResolvedExtensions } from '@openshift-console/dynamic-plugin-sdk';
 import NetflowTraffic from "../netflow-traffic";
+import { extensionsMock } from "../__tests-data__/extensions";
+
+jest.mock('@openshift-console/dynamic-plugin-sdk', () => ({
+  useResolvedExtensions: jest.fn(),
+}));
+const useResolvedExtensionsMock = useResolvedExtensions as jest.Mock;
 
 describe("<NetflowTraffic />", () => {
-  it('should render component', () => {
+  beforeAll(() => {
+    useResolvedExtensionsMock.mockReturnValue(extensionsMock);
+  });
+
+  it('should shallow component', () => {
     const wrapper = shallow(
       <NetflowTraffic />
     );
     expect(wrapper.find(NetflowTraffic)).toBeTruthy();
   });
+  it('should render toolbar components', () => {
+    const cheerio = render(
```

Comment:
out of curiosity, why that name? (cheerio)

Reply:
actually the `render` function returns an implementation of jquery based on [cheeriojs](https://github.com/cheeriojs/cheerio) whereas `shallow` and `mount` returns a `ShallowWrapper` that extends react `Component` so you don't have the same functions calls